third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/TargetInfo.cpp

```diff
@@ -493,6 +494,12 @@ bool TargetInfo::canUseStMatrix(RankedTensorType tensorTy,
   if (order[0] != 1)
     return false;
 
+  // Each chunk is filled in with a single warp
+  int numColsPerChunk = (8 * swizzleByteSize) / getElementBitWidth(tensorTy);
+  int instrN = mmaLayout.getInstrShape()[1];
+  if (instrN < numColsPerChunk)
```

Comment:
This is to avoid the assert on line 1215 in `LinearLayoutConversions.cpp`.

Reply:
Thanks. 
FYI, this was still a workaround, we should be able to use multiple warps in a chunk by a follow up cleanup